functions/_nvm_use_on_pwd_change.fish

```diff
@@ -0,0 +1,9 @@
+function _nvm_use_on_pwd_change \
+    --on-variable nvm_use_on_pwd_change \
+    --on-variable PWD \
+    --description 'Use Node.js version specified by project automatically'
+
+    if test "$nvm_use_on_pwd_change" = true
```

Comment:
Also reasonable would be to check if the variable is just set. Maybe that's even fishier.